extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -250,6 +256,13 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     super.onCreativeRender(isVerifiedAmpCreative);
     injectActiveViewAmpAnalyticsElement(
         this, this.ampAnalyticsConfig, this.responseHeaders_);
+    const iframe = this.element.querySelector('iframe');
```

Comment:
One thing that occurred to me is that amp-analytics is measuring the viewability of the amp-ad element as opposed to the frame within it.  Given we always center the creative both vertically and horizontally this should be fine.

I'm actually thinking we should really consider this a variant of multisize support where we can just return the size in extractCreativeAndSignature.  Wouldn't that essentially handle this case without the need to make any other changes (e.g. it already centers)?